tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -23,10 +24,17 @@ internal static class WafConfigurator
 
         internal static InitializationResult Configure(string rulesFile, WafNative wafNative, Encoder encoder, string obfuscationParameterKeyRegex, string obfuscationParameterValueRegex)
         {
+            if (wafNative.ExportErrorHappened)
+            {
+                Log.Error("Waf couldn't initialize properly because of missing methods in native library, please check the version of libddwaf");
```

Comment:
nit: I wonder, can we be more explicit about _which_ version we expect here? Probably not 🤔 I think at the least we should give some specific guidance. e.g. make sure you don't have multiple versions of Datadog.Trace, install the latest, etc

Reply:
we cant, unless we build some kinda compatibility matrice of tracer versions / waf versions.. 
I added more guidance in the comment in https://github.com/DataDog/dd-trace-dotnet/pull/3251/commits/9fe3b5df8b64a610153f2a3ef1677b24a138514d